cmd/keymasterd/config.go

```diff
@@ -246,6 +245,12 @@ func loadVerifyConfigFile(configFilename string) (RuntimeState, error) {
 		runtimeState.Config.SymantecVIP.Client = &client
 	}
 
+	//
+	if runtimeState.Config.Base.HideStandardLogin == true && runtimeState.Config.Oauth2.Enabled == false {
```

Comment:
This is not idiomatic. This is the preferred way:
`if runtimeState.Config.Base.HideStandardLogin && !runtimeState.Config.Oauth2.Enabled {`